torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -829,7 +829,8 @@ def special_pattern_replacement(model: QuantizedGraphModule):
         is_call_function, is_call_method, is_call_module = is_special_pattern_node(ref_node, modules)
         if not (is_call_module or is_call_function or is_call_method):
             continue
-        dq_node_or_nodes = ref_node.args[0]
+        assert len(ref_node.args) > 0 or len(ref_node.kwargs) > 0
```

Comment:
if it works for your case, maybe we can start with requiring len(ref_node.kwargs) == 1

Reply:
for `torch.cat`, there would be two kwargs, `tensors` and `dim`. It may not work if we limit the number to be 1